src/main/java/core/basesyntax/repoter/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.repoter;
+
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String HEADER = "fruit,quantity";
+    private static final String DEFAULT_DELIMITER = ",";
+    private StringBuilder builder;
+
+    public ReportGeneratorImpl() {
+        builder = new StringBuilder();
+    }
```

Comment:
Since the 'builder' is suggested to be a local variable, the constructor and its initialization of 'builder' are no longer necessary.